devel/buildbot-worker/files/org.macports.buildbot-worker.template.plist

```diff
@@ -17,7 +17,7 @@
 		<string>--nodaemon</string>
 		<string>--no_save</string>
 		<string>--python=buildbot.tac</string>
-		<string>--pidfile=buildslave.pid</string>
+		<string>--pidfile=twistd.pid</string>
```

Comment:
This could potentially be changes on the master as well, but there's no real need.
I just wanted to change this to get rid of the obsolete reference to "slave".